### PR TITLE
OEV gateway

### LINKED ISSUE #1215: Implement the HTTP gateway to facilitate OEV auctions
Documentation for https://github.com/api3dao/airnode/issues/1579

### PR DESCRIPTION
**Pull request by amarthadan**
Close #1215

I've added a separate page for the OEV gateway as the purpose of this gateway is quite different than the rest of the gateways. But feel free to rearrange it as you see fit if you feel differently.

Jacob (can't tag you for some reason, will DM you) feel free to add more details but it feels like it's not really necessary on the Airnode side and anything else important will be part of the OEV relay docs.

### CODE REVIEW

**Comment by jgcogsystematictrading** [comment]
It looks fine, my only thought was maybe you could expand on what is included in the returned signature from the gateway but probably not be too important.

File: docs/airnode/v0.11/grp-providers/guides/build-an-airnode/oev-gateway.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+- `bidderAddress` - Blockchain address of the winning searcher.
+- `bidAmount` - Bid amount that won the auction.
+- `signedData` - A list of beacon data to be signed. It can contain two types:
+  full beacon data, only beacon metadata
+- `airnodeAddress` - Airnode address identifying a beacon
+- `endpointId` - Endpoint ID identifying a beacon
+- `encodedParameters` - Parameters in their encoded form identifying a beacon
+- `timestamp` - UNIX timestamp of the beacon data
+- `encodedValue` - Beacon value in its encoded form
+- `signature` - Signature of the beacon data
```
Comment (dcroote) [inline docs/airnode/v0.11/grp-providers/guides/build-an-airnode/oev-gateway.md]: I would indent these bullets given they belong within `signedData`.
```suggestion
  - `airnodeAddress` - Airnode address identifying a beacon
  - `endpointId` - Endpoint ID identifying a beacon
  - `encodedParameters` - Parameters in their encoded form identifying a beacon
  - `timestamp` - UNIX timestamp of the beacon data
  - `encodedValue` - Beacon value in its encoded form
  - `signature` - Signature of the beacon data
```

**Review by dcroote — APPROVED** [review]
LGTM, one minor suggestion

**Comment by wkande** [comment]
@dcroote @amarthadan Should this be hidden until OEV is actually functional, or maybe it is. It is a feature that by having presence in the docs, readers might believe is something that can implement. There are no OEV auctions right now if I am not mistaken. Do I not understand?

**Comment by amarthadan** [comment]
You are correct, there are no auctions at the moment. But this is not really a user-facing piece anyway. I think it's ok to include it, it's an Airnode feature. But nobody will be actually able to use it really :man_shrugging:. @jgcogsystematictrading WDYT?

**Pull request merged**